lib/mongo-cache.js

```diff
@@ -44,7 +44,7 @@ function makeTileName(req) {
   return req.path + queryString;
 }
 
-function setupQuery(survey, tile) {
+function setupQuery(survey, tile, deleted) {
```

Comment:
How about using an options object with a `selectDeleted` property? Three params is my informal trigger for switching to options.